actions/lib/formatters.py

```diff
@@ -118,3 +126,20 @@ def to_attachment_dict(attachment):
         'content': attachment.content,
     }
     return result
+
+
+def to_links_dict(issue):
+    """
+    :rtype: ``dict``
+    """
+    result = {
```

Comment:
Minor nit, you can probably just `return` direct here instead of assigning to `result`. But fine if we want to leave that as-is.

Reply:
I just mimicked the style of the other functions.  I'll leave it as-is just to be consistent if that's ok.

Reply:
Not a hill i will die on, thats fine.